extension/manifest.json

```diff
@@ -6,41 +6,15 @@
       "update_url": "https://addons.firefox.com.cn/chinaedition/addons/updates.json?reqVersion=%REQ_VERSION%&id=%ITEM_ID%&version=%ITEM_VERSION%&maxAppVersion=%ITEM_MAXAPPVERSION%&status=%ITEM_STATUS%&appID=%APP_ID%&appVersion=%APP_VERSION%&appOS=%APP_OS%&appABI=%APP_ABI%&locale=%APP_LOCALE%&currentAppVersion=%CURRENT_APP_VERSION%&updateType=%UPDATE_TYPE%&compatMode=%COMPATIBILITY_MODE%"
     }
   },
-  "background": {
-    "scripts": [
-      "newtab/background.js"
-    ]
-  },
-  "chrome_url_overrides": {
-    "newtab": "newtab/newtab.html"
-  },
+  "background": {},
   "default_locale": "en_US",
   "description": "__MSG_extensionDescription__",
-  "experiment_apis": {
-    "chinaNewtab": {
-      "schema": "mozillaonline/schemas/china_newtab.json",
-      "parent": {
-        "scopes": ["addon_parent"],
-        "script": "mozillaonline/parent/ext-chinaNewtab.js",
-        "events": ["startup"],
-        "paths": [
-          ["mozillaonline", "chinaNewtab"]
-        ]
-      }
-    }
-  },
   "homepage_url": "http://www.firefox.com.cn/",
   "icons": {
     "48": "icons/icon-48.png",
     "64": "icons/icon-64.png"
   },
   "manifest_version": 2,
   "name": "__MSG_extensionName__",
-  "permissions": [
-    "mozillaAddons",
-    "storage",
-    "telemetry",
-    "https://*.firefoxchina.cn/*"
-  ],
-  "version": "5.114"
+  "version": "6.0"
```

Comment:
This alone is not enough, it will be overriden by the value in `package.json`, see also 4c2ab8f6